compose/cli/main.py

```diff
@@ -563,8 +563,14 @@ def images(self, options):
             rows = []
             for container in containers:
                 image_config = container.image_config
+                service = self.project.get_service(container.service)
+                if service.image_name in image_config['RepoTags']:
+                    index = image_config['RepoTags'].index(service.image_name)
+                else:
+                    index = 0
```

Comment:
nit: move `index = 0` before the `if` branch and get rid of the `else` entirely.